### DISCUSSION TITLE
Waveshare RS485

### DISCUSSION THREAD

**Discussion by BaasHarm** [wills106/homeassistant-solax-modbus#122] [Q&A]
I'm eager to try this out so I can get rid of the solax cloud integration.
I happen to have this laying around:

https://www.waveshare.com/rs485-to-eth.htm

Would that work with my X1 mini (X1-1.5-S-D)?

**Comment by infradom** [comment]
I only have experience with the rail mounted waveshare device. See our wiki page for more information
![image](https://user-images.githubusercontent.com/11804014/191832203-489a6e80-11f0-4252-8ed3-72dc178a645a.png)
